stdlib/InteractiveUtils/src/InteractiveUtils.jl

```diff
@@ -6,7 +6,7 @@ module InteractiveUtils
 
 export apropos, edit, less, code_warntype, code_llvm, code_native, methodswith, varinfo,
     versioninfo, subtypes, peakflops, @which, @edit, @less, @functionloc, @code_warntype,
-    @code_typed, @code_lowered, @code_llvm, @code_native, Pkg, clipboard
+    @code_typed, @code_lowered, @code_llvm, @code_native, clipboard
```

Comment:
The goal here is make it such that `Pkg` is available by default at the REPL

Reply:
Ok, I can add a `using Pkg` to the repl startup code in the same place where we do `using InteractiveUtils`.